applications/targets/tests/test_services.py

```diff
@@ -16,20 +16,29 @@ def call_matching_service(self):
         matching = TargetMatchingService()
         matching.process_target(self.target)
 
+    @mock.patch(
+        'applications.chats.services.RoomCreator.create_from_target_match',
+        return_value=None
+    )
```

Comment:
If it's the same mocks for all tests can't you do it at the class level instead?